operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -241,13 +241,20 @@ if [ ! "${SERVER_NAME}" = "introspector" ]; then
     [ ! $? -eq 0 ] && trace SEVERE "Could not remove stale file '$wl_state_file'." && exit 1
   fi
 
+  if [ -f /weblogic-operator/introspector/domainzip.secure ]; then
```

Comment:
This check looks a little obscure/fragile to me. Instead, how about checking the DOMAIN_SOURCE_TYPE env var? (You'll need to make sure that this env var is passed to all servers - the operator might only be passing it to the introspector pod right now...)